File: api/src/test/java/work/beltran/discogsbrowser/api/WantedListTest.java
package work.beltran.discogsbrowser.api;

import org.junit.Test;

import rx.observers.TestSubscriber;
import work.beltran.discogsbrowser.api.model.UserWanted;

/**
 * Created by Miquel Beltran on 8/27/16
 * More on http://beltran.work
 */
public class WantedListTest extends DiscogsServiceTest {

    private static final String USER = "mike513";
    private static final int PAGE = 0;

    @Test
    public void testWantedList() throws Exception {
        TestSubscriber<UserWanted> subscriber = TestSubscriber.create();
        service.getWantedList(USER, PAGE).subscribe(subscriber);
        subscriber.assertNoErrors();
        subscriber.assertCompleted();
    }
}

File: api/src/test/java/work/beltran/discogsbrowser/api/WantedListTest.kt
package work.beltran.discogsbrowser.api

import org.junit.Test

/**
 * Created by Miquel Beltran on 8/27/16
 * More on http://beltran.work
 */
class WantedListTest : DiscogsServiceTest() {

    @Test
    fun `get wanted list`() {
        val subscriber = service.getWantedList(USER, PAGE).test()
        subscriber.assertNoErrors()
        subscriber.assertComplete()
    }

    companion object {
        private const val USER = "mike513"
        private const val PAGE = 0
    }
}
